pkgdown/_brand.yml

```diff
@@ -0,0 +1,29 @@
+meta:
+  name: bslib
+  link:
+    home: https://rstudio.github.io/bslib
+    github: https://github.com/rstudio/bslib
+
+color:
+  info: "#e6f2fd"
+
+typography:
+  fonts:
+    - family: Atkinson Hyperlegible
+      source: google
+      weight: [300, 400, 700]
+  base: Atkinson Hyperlegible
+  headings:
+    color: "#206B72"
+  link:
+    color: "#216b72"
+    decoration: underline dotted
+
+defaults:
+  bootstrap:
+    defaults:
+      link-color-dark: "#cef4f9"
```

Comment:
This is a good argument for `color` and `background-color` to take light/dark variants in brand.yml (something we'll think about doing in the future).

```yml
typography:
  link:
    color:
      light: "#216b73"
      dark: "#cef4f9"
```